src/cdk/drag-drop/drag-drop-registry.ts

```diff
@@ -17,6 +29,23 @@ const activeCapturingEventOptions = normalizePassiveListenerOptions({
   capture: true,
 });
 
+/** Keeps track of the apps currently containing badges. */
```

Comment:
nit: change "badges" to be about drag-and-drop

Reply:
This _definitely_ didn't happen because the implementation is the same as the badge...